src/borg/testsuite/platform.py

```diff
@@ -44,6 +43,7 @@ def fakeroot_detected():
 
 def user_exists(username):
     try:
+        import pwd # buildin but not on all OS
```

Comment:
so doesn't it still cause a problem when this function is called (on windows)?

Reply:
The Problem here ist that this is used in the unittest.attribute and otherwise this produced some circular refs.

(A very bad idea to find a user via his user directory... may there are better ways)